test/acceptance/audio-upload-composer-test.js

```diff
@@ -0,0 +1,174 @@
+import {
+  click,
+  find,
+  settled,
+  visit,
+  waitFor,
+  waitUntil,
+} from "@ember/test-helpers";
+import { test } from "qunit";
+import {
+  acceptance,
+  query,
+  visible,
+} from "discourse/tests/helpers/qunit-helpers";
+import I18n from "discourse-i18n";
+
+const uploadResponseData = {
```

Comment:
@Arkshine Another small note for future reference, that this constant should probably be in a "fixture".

Reply:
Noted!